include/server/ErrorMessage.h

```diff
@@ -0,0 +1,120 @@
+/**
+ *  \file ErrorMessage.hpp
```

Comment:
now it doesn't build because the other files still `#include "ErrorMessage.h"` :stuck_out_tongue_winking_eye: